### PR TITLE
Fixing saving last tank bug

### LINKED ISSUE #478: Heading from training arena back to casus sometimes switches tanks
**Describe the bug**
It is rare, but sometimes when I am heading back to casus from the training arena, I get a different tank.

**To Reproduce**
1. Go to training arena through casus.
2. Go back to casus.

**Expected behavior**
It should retain the cookie, I haven't changed my selected tank in the training arena

**Screenshots**
N/A

**Additional context**
I need to test this more to effectively reproduce.

### PR DESCRIPTION
**Pull request by SecondThread**
**Description**
Fixed the bug with the casus tank getting reset after a match. There was also another bug where the tank you set was not getting saved correctly in Armory. This was because we were using sentinal values and passing around non-null tanks, so the cookie was getting cleared. I fixed that here, so now thing should work correctly.

image

**Resolves These Issues**
Resolves #478 

**Type of change**
Check options that are relevant.

- [x] Bug fix (fixes a bug issue)
- [ ] New feature (adds functionality)
- [ ] This fix or feature will cause existing functionality to not work as expected. (Please elaborate in Description.)

**Steps to Verify Functionality**
Watch a match to finish in the training arena, and then immediately click "back to casus". You should consistently be on the correct tank now.

**Final Checklist:**
Go down the list and check them off.

- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have performed a self-review of my own code
- [x] My changes pass Flow, build without errors and (if applicable) pass Jest tests.
- [ ] This change requires a update in the design document (if applicable)

### CODE REVIEW

File: src/battleground/Battleground.js [hunk 1]
```diff
@@ -55,7 +55,7 @@ const arenaWidth: {[ArenaType]: number} = {
 }
 
 const matchLengthForArena: {[ArenaType]: number} = {
-	DIRT: INTRO_LENGTH + 30*60,
+	DIRT: INTRO_LENGTH + 30*1,
```
Comment (Derplime) [inline src/battleground/Battleground.js]: Is this supposed to be different from the rest?

Comment (Derplime) [inline src/battleground/Battleground.js:58]: Is this supposed to be different from the rest?

**Review by Derplime — APPROVED** [review]
I have one comment included otherwise it's good.

File: src/battleground/Battleground.js (hunk 1 above)
Comment (SecondThread) [reply src/battleground/Battleground.js]: oops, no I just had that in there for testing so I didn't have to wait for the whole minute. Nice catch.

**Pull request merged**